config/webpack.config.web.js

```diff
@@ -19,8 +21,39 @@ const getClientEnvironment = require("./env");
 
 // Source maps are resource heavy and can cause out of memory issue for large source files.
 const shouldUseSourceMap = process.env.GENERATE_SOURCEMAP !== "false";
+
+function loadRepoRegistry() /*: RepoIdRegistry */ {
+  const env = process.env.SOURCECRED_DIRECTORY;
+  // TODO(#945): de-duplicate finding the directory with src/cli/common.js
+  const defaultDirectory = path.join(os.tmpdir(), "sourcecred");
+  const scDirectory = env != null ? env : defaultDirectory;
+  // TODO(@dandelion): Remove hacks around compat usage here
+  // TODO(@dandelion): Import rather than hardcode the registry file name
```

Comment:
Yep. For posterity: this is all a bit hacky because most of our modules
are written with ES6 syntax, but our Webpack config needs to run in
vanilla Node, so we’re inlining things left and right.